pkg/controllers/cloudentityselector/controller.go

```diff
@@ -145,3 +151,23 @@ func (r *CloudEntitySelectorReconciler) processDelete(selectorNamespacedName *ty
 	_ = r.AccManager.RemoveResourceFiltersFromAccount(&accountNamespacedName, selectorNamespacedName)
 	return nil
 }
+
+// setStatus sets the status on the CloudEntitySelector CR.
+func (r *CloudEntitySelectorReconciler) setStatus(namespacedName *types.NamespacedName, err error) {
+	var errorMsg string
+	if err != nil {
```

Comment:
wont this be called if there is an error. If its nil, do you want to respond? Or you are using this to clear error?